cmd/podman/containers/create.go

```diff
@@ -154,3 +159,18 @@ func pullImage(imageName string) error {
 	}
 	return nil
 }
+
+func convertDevices(cmd *cobra.Command) ([]specs.LinuxDevice, error) {
```

Comment:
i believe this has to be done in the specgen/generate of container create to account for actually being on the server?

Reply:
or is the path sufficient? if so, then this probably belongs in the area where the specgen is created based on the []string list of devices.

Reply:
There is no verification going on here.  But specgen insists on this list being a []specs.LinuxDevice.

Reply:
Simplified this one a lot.  PTAL